web/src/components/historySection.js

```diff
@@ -0,0 +1,105 @@
+import React, { useState, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import loadable from '@loadable/component';
+import { slidesToShowPlugin } from '@brainhubeu/react-carousel';
+
+import HistoryItem from './historyItem';
+
+const Carousel = loadable(() => import('./carouselWrapper'));
+
+// const isBrowser = typeof window !== 'undefined';
+
+export default function History({ years }) {
+    const [activeTab, setActiveTab] = useState(years[0].id);
+    const [buttonComponentLogic, setButtonComponentLogic] = useState(false);
+
+    const setHistoryTimeline = () => {
+        if (window.innerWidth < 600) {
+            setButtonComponentLogic(true);
+        }
+    };
+
+    useEffect(() => {
+        setHistoryTimeline();
+        window.addEventListener('resize', setHistoryTimeline);
+
+        return () => {
+            window.removeEventListener('resize', setHistoryTimeline);
+        };
+    }, []);
+
+    return (
+        <section className="container my-24">
+            <h2 className="uppercase text-ts-h2 mb-4">History</h2>
+            <div>
+                <div className="flex gap-20">
+                    {buttonComponentLogic ? (
+                        <Carousel
+                            plugins={[
+                                'fastSwipe',
```

Comment:
What does `fastSwipe` do?

Reply:
@asasmith changes the swiping behavior

Reply:
What does it change?

Reply:
the way it scrolls, it's faster @asasmith

Reply:
I don’t see anything in the docs about it affecting scroll speed. I’d like to avoid adding things if it’s not clear what they’re doing or what value they bring.